app/http/routes/status.go

```diff
@@ -0,0 +1,52 @@
+package routes
+
+import (
+	"net/http"
+
+	"github.com/acouvreur/sablier/app/instance"
+	"github.com/acouvreur/sablier/app/sessions"
+	"github.com/acouvreur/sablier/config"
+	"github.com/gin-gonic/gin"
+)
+
+type Status struct {
+	SessionsManager sessions.Manager
+	StrategyConfig  config.Strategy
+	SessionsConfig  config.Sessions
+}
+
+func NewStatus(sessionsManager sessions.Manager, strategyConf config.Strategy, sessionsConf config.Sessions) *Status {
+
+	status := &Status{
+		SessionsManager: sessionsManager,
+		StrategyConfig:  strategyConf,
+		SessionsConfig:  sessionsConf,
+	}
+
+	return status
+}
+
+type StatusResponse struct {
+	ManagedServices []instance.State `json:"managed_services"`
+}
+
+func (s *Status) ServeStatus(c *gin.Context) {
+	containers, _ := s.SessionsManager.GetManagedContainers()
```

Comment:
It's not always containers, it might be swarm services, kubernetes deployments, etc.
We should use `instances` instead.

Reply:
Makes sense here. I only have docker experience. I will look into using instances.